### DISCUSSION TITLE
milvusbootcamp/pic-search-webserver:0.1.0  grpc StatusCode.UNIMPLEMENTED

### DISCUSSION THREAD

**Discussion by dillonzyl** [milvus-io/milvus#6138] [Q&A and General discussion]
<_Rendezvous of RPC that terminated with:
	status = StatusCode.UNIMPLEMENTED
	details = ""
	debug_error_string = "{"created":"@1624524747.603951219","description":"Error received from peer ipv4:192.168.2.99:19530","file":"src/core/lib/surface/call.cc","file_line":1055,"grpc_message":"","grpc_status":12}"
>

**Comment by JackLCL** [comment]
Which document are you following? Can you post a link to it?
